### PR TITLE
ffmpeg: Add zimg dependency

### PR DESCRIPTION
**Pull request by Biswa96**
As requested here https://github.com/msys2/MINGW-packages/issues/10637

### CODE REVIEW

File: mingw-w64-ffmpeg/PKGBUILD [hunk 1]
```diff
@@ -136,7 +138,12 @@ build() {
     --disable-doc
   )
 
-  if [[ ${MINGW_PACKAGE_PREFIX} != *-clang-* ]]; then
+  if [[ ${MINGW_PACKAGE_PREFIX} == *-clang-* ]]; then
+    common_config+=(
+      --cc=clang
+      --cxx=clang++
+    )
+  else
```
Comment (lazka) [inline mingw-w64-ffmpeg/PKGBUILD:146]: We do set CC and CXX now, so in theory you could pass `--cc=$CC -cxx=$CXX` unconditionally

Comment (Biswa96) [reply mingw-w64-ffmpeg/PKGBUILD:146]: OK, I will add that. Accidentally, that was my another choice. But then I choose to set the options explicitly to make it future proof. Things are changing a lot now-a-days 😅

Comment (lazka) [reply mingw-w64-ffmpeg/PKGBUILD:146]: rolling ~~distro~~breakage

Comment (Biswa96) [reply mingw-w64-ffmpeg/PKGBUILD]: Done.

**Pull request merged**